lesson7/airflow/dags/process_sales.py

```diff
@@ -0,0 +1,52 @@
+from airflow.decorators import dag
+from airflow.decorators import task
+from airflow.exceptions import AirflowFailException
+from datetime import datetime
+import requests
+import os
+
+os.environ["no_proxy"] = "*"
+GET_API_URL = 'http://127.0.0.1:8081/apiworker'
+CONVERT_URL = 'http://127.0.0.1:8081/storageworker'
+
+
+@dag(
+    schedule='0 1 * * *',
+    start_date=datetime(2022, 8, 9),
+    end_date=datetime(2022, 8, 12),
+    catchup=True,
+    max_active_runs=1
+    )
+def process_sales():
+    @task
+    def extract_data_from_api(**kwargs):
```

Comment:
I expect students to use SimpleHttpOperator here. But this way works for this task, too
In a real project, SimpleHttpOperator allows to use authentication and keeping credentials in connections.